Formula/sonar-scanner.rb

```diff
@@ -0,0 +1,29 @@
+class SonarScanner < Formula
+  desc "Launcher to analyze a project with SonarQube"
+  homepage "http://docs.sonarqube.org/display/SONAR/Analyzing+with+SonarQube+Scanner"
```

Comment:
Looks like this redirects to `http://docs.sonarqube.org/display/SCAN/Analyzing+with+SonarQube+Scanner` so we may as well use that.